axiom/nr_txn.h

```diff
@@ -471,6 +471,17 @@ extern nr_status_t nr_txn_record_error_worthy(const nrtxn_t* txn, int priority);
 #define NR_TXN_ALLOW_RAW_EXCEPTION_MESSAGE \
   "Message removed by New Relic security settings"
 
+extern void nr_txn_record_error_with_additional_attributes(nrtxn_t* txn,
```

Comment:
Could you document the record error functions (both forms)?

Reply:
Added: [c5a40cc](https://github.com/newrelic/newrelic-php-agent/pull/942/commits/c5a40cc3cabb46abb889e5ba51355091dd73f2e2)